src/compiler/checker.ts

```diff
@@ -1456,6 +1457,11 @@ export function createTypeChecker(host: TypeCheckerHost): TypeChecker {
     const requireSymbol = createSymbol(SymbolFlags.Property, "require" as __String);
     const isolatedModulesLikeFlagName = compilerOptions.verbatimModuleSyntax ? "verbatimModuleSyntax" : "isolatedModules";
 
+    const InfinitySymbol = createSymbol(SymbolFlags.Property, "Infinity" as __String);

```

Comment:
Prefer to not capitalize these, since they're not classes (even if it makes them a bit less consistent with their symbol names, it's fine).